1001-2000/LIT1933Mashaf.xml

```diff
@@ -10,6 +10,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <title xml:lang="ar" xml:id="t2arabic">كتاب الحاوي</title>
             <title xml:lang="ar" corresp="#t2arabic" type="normalized">Kitāb al-ḥāwi</title>
```

Comment:
Although not part of your changes, maybe you can correct it to "ḥāwī", now that you are working on the file